_partials/_cloud-mst-comparison.md

```diff
@@ -3,11 +3,12 @@
   header="Timescale Cloud or MST?"
 >
 
-Timescale Cloud is a high-performance developer focused cloud that provides PostgreSQL services enhanced 
-with our blazing fast vector search. You can securely integrate Timescale Cloud with your AWS, GCS or Azure 
-infrastructure. [Create a Timescale Cloud service][timescale-service] and try for free.
+Both the Managed Service for TimescaleDB (MST) by Aiven and Timescale Cloud are TimescaleDB services built on 
+PostgreSQL for relational, time-series, vector search, and more. MST is a fully managed TimescaleDB service while 
```

Comment:
Both TSC and MSC supply fully hosted services. Both can be used for development. TSC has all the latest Timescale stuff and groovy things that MSC does not have. MSC runs on GCP and Azure, TSC on AWS.  Can we think up another line to differentiate please.

Reply:
> Both TSC and MSC supply fully hosted services. Both can be used for development. TSC has all the latest Timescale stuff and groovy things that MSC does not have

Let's not frame it this way. Let's just say they include different features and that Timescale Cloud runs on AWS and MST runs on GCP and Azure.